string mode
string robot_name

---

int64 code
string message
